src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.js

```diff
@@ -144,6 +144,20 @@ const MetaList = () => {
     [handlingTimesBySlug, incident]
   );
 
+  const [processTimeCopy, processTimeClass] = useMemo(() => {
```

Comment:
If you'd name this `processTimeText` instead of `processTimeCopy` that would align with naming conventions in other parts of the app